### DISCUSSION TITLE
AWS deployment instructions

### DISCUSSION THREAD

**Discussion by A5HRAJ** [mxpv/podsync#268] [General]
Can anyone post detailed instructions for AWS deployment? I have an instance of Podsync running, but nothing loads when I go to its address.

**Comment by dchakarov** [comment]
I have the same issue - I got "create complete" on AWS, I see a feed URL in Outputs and when I try to open it, I get Failed to open URL. Please advise.

**Reply by ghost** [reply]
I just made it work, here some hints which might help you if you are a little bit familiar with AWS:
- You can edit the podsync config by downloading/editing/uploading `cloud_formation.yml` when you launch the stack (first step); this helped me to tune some download timeout parameters and its also more convenient to configure the feeds there (instead of having just a single one configurable via the stack params)
- To debug, you can connect into the EC2 instance by going to the resources tab in the deployed stack; then use the EC2 view to connect into the instance; then use `docker ps` and `docker logs -f <containerId>` to print out the log
- Use the log output to determine whether podsync is even able to fetch episodes
- Remove the path of link in the outputs of the stack (PodsyncUrl) to check whether the RSS xml is even generated (i. e. `http://ec2-abcdef.us-west-1.compute.amazonaws.com:8080/` should list some XML and subfolders)
